hbase-client/src/main/java/org/apache/hadoop/hbase/shaded/protobuf/RequestConverter.java

```diff
@@ -590,7 +590,9 @@ public static void buildNoDataRegionActions(final byte[] regionName,
     if (!multiRequestBuilder.hasNonceGroup() && hasNonce) {
       multiRequestBuilder.setNonceGroup(nonceGroup);
     }
-    multiRequestBuilder.addRegionAction(builder.build());
+    if (builder.getActionCount() > 0) {
```

Comment:
What will happen for server side if empty RegionAction added?

Reply:
Thank you for taking a look at this!

Actually the server side does nothing as the RegionAction is empty. So I'd say it's not a critical bug. It will just waste network traffic a bit I think.